### ISSUE TITLE
capture rangeBegin and rangeEnd for response videos

### ISSUE THREAD

**Issue by smurp** [DIVERSUSandTIM/priorities#13]
## the semantic aspects, storing and retrieving data
* [x] obtain the data from UX
* [x] inject the data into the triples
* [ ] ensure Begin is before End

## the UX aspects, obtaining and displaying the data
* [x] figure out how we want users to express the start and stop times or no time
* [x] implement them
* [ ] show the moving triangles under main video, synced to rangeBegin and rangeEnd
* see #26 for how to display the rangeBegin and rangeEnd data

@SamXBauer @wolfmaul

**Comment by smurp** [comment]
OK. So there are now `rangeBegin` and `rangeEnd` form fields which are automatically updated by the player (with 100th of a second precision).  Their value is captured at form submission time and injected into the `medifrag:` value. 

Adding a lock/unlock feature so a person has an alternative to just manually editing the values....

**Comment by SamXBauer** [comment]
start and stop times: this is the range to which the user refers in the anchor video?

**Comment by smurp** [comment]
@SamXBauer Yeah, working now.  Play the main video and watch them change.  Lock start and stop when you want.

## Proposals:

#### Autoloop
Presss play (on the main video) and the Begin and End values track the time continuously (as now) then:
* user click lock on Begin
* user clicks lock on End 
* then the video playback loops immediately back to Begin and plays over again, looping between begin and end

#### Arrow icons
There should be icons like &#9698; and &#9699; right under the main video which show the Begin and End points graphically.  In the flower this would probably be something like ┣━━┫curved around the perimeter.

#### Unlocked Begin Pushes Locked End

*Problem:*
 
When End is locked and Begin is not and the video is playing, then Begin can *pass* End causing nonsense values. 

*Solution:*

Once an unlocked Begin value (pushed along by a playing video) bumps into a locked End value, have the value of End remain "locked" but have it grow to stay equal to the climbing value of Begin.  Have it *pushed along* by the Begin value.
